dl/src/main/scala/com/intel/analytics/bigdl/dataset/Transformer.scala

```diff
@@ -70,66 +72,162 @@ class Identity[A] extends Transformer[A, A] {
   }
 }
 
+
 object SampleToBatch {
   def apply[T: ClassTag]
-  (batchSize: Int)
+  (batchSize : Int,
+   padFeatureValue : Option[Tensor[T]] = None,
+   padLabelValue : Option[T] = None,
+   fixLength: Option[Int] = None)
```

Comment:
please add comments and scala doc; change the arguments to featurePadding, lablePadding, and fixedLength.